src/main/target/ALIENWHOOP/target.h

```diff
@@ -184,7 +184,7 @@
  */
 /* Assume Spektrum following defines inherited from common_fc_pre.h:
 //#define USE_SERIALRX_SPEKTRUM
-//#define USE_SPEKTRUM_BIND
+//#define SPEKTRUM_BIND
```

Comment:
This change is within a comment and won't have any effect on the ALIENWHOOP target. I have made note of the correct define and will correct in future PR. Actually, I will probably remove all of those comments. 😄

Reply:
Good point. I use a regex to find / replace the defines, and I've included commented out defines to preventatively change commented out defines.

Reply:
And I can always duck behind the 'I used a regex, it *must* be correct. :stuck_out_tongue_closed_eyes: 

But thanks for your trust!